main.go

```diff
@@ -5,9 +5,30 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"strings"
+
+	"github.com/google/uuid"
 )
 
-func getPing(w http.ResponseWriter, r *http.Request) {
+type Book struct {
+	Name      string    `json:"name"`
+	Price     float32   `json:"price"`
+	Inventory int       `json:"inventory"`
+	ID        uuid.UUID `json:"id"`
```

Comment:
Lets move the ID to the first line in the struct definition.

Reply:
This is really necessary? Because the json.NewDecoder allocates the message received (with name, price and inventory; without ID) in the first 3 fields of the Book struct. I've tried to change the ID to the first line and the NewDecoder get lost.
In the database table I've put the ID in the last field too, even defining it as primary key.

Reply:
Json Marshal getting lost is not a thing. I am sure that it is some other issue, if you are available we can talk today.

Reply:
Changed. ID is the first.